### PR TITLE
Add a NAME/USDC to spot index mapping to the coin_to_asset HashMap

### PR DESCRIPTION
**Pull request by Frixoe**
To allow spot ordering with NAME/USDC when placing spot orders

### CODE REVIEW

File: src/meta.rs [hunk 1]
```diff
@@ -12,6 +14,40 @@ pub struct SpotMeta {
...
+            .collect();
+
+        for asset in self.universe.iter() {
+            let spot_ind: u32 = 10000 + asset.index as u32;
+            let name_to_ind = (asset.name.clone(), spot_ind);
+
+            let token_1_name = if let Some(name) = index_to_name.get(&asset.tokens[0]) {
+                name
+            } else {
+                "" // Keeping these empty because the user won't be able to match then so no harm
```
Comment (lmlmt) [inline src/meta.rs:35]: nit: just continue

also use `let/else` for this:
```
let Some(token1_name) = index_to_name.get(&asset.tokens[0]) else {
  continue;
};
```

File: src/meta.rs [hunk 2]
```diff
@@ -12,6 +14,40 @@ pub struct SpotMeta {
...
+            };
+            let token_2_name = if let Some(name) = index_to_name.get(&asset.tokens[1]) {
+                name
+            } else {
+                ""
```
Comment (lmlmt) [inline src/meta.rs:40]: ditto

File: src/meta.rs (hunk 1 above)
Comment (Frixoe) [reply src/meta.rs]: Good idea. Updated, thanks.

File: src/meta.rs (hunk 2 above)
Comment (Frixoe) [reply src/meta.rs]: Done the same here too

**Review by lmlmt — APPROVED** [review]

**Pull request merged**